Documentation/gettingstarted_kafka.rst

```diff
@@ -0,0 +1,392 @@
+
+    $ ./kafka-consume.sh --topic empire-announce
+
+Do the same in the *outpost-9999* window:
+
+::
+
+    $ ./kafka-consume.sh --topic empire-announce
+
+Now from the *empire-hq*, first produce a message to the *empire-announce* topic:
+
+::
+
+   $ echo "Happy 40th Birthday to General Tagge" | ./kafka-produce.sh --topic empire-announce
+
+This message will be posted to the *empire-announce* topic, and shows up in both the *outpost-8888* and
+*outpost-9999* windows who consume that topic.   It will not show up in *empire-backup*.
+
+*empire-hq* can also post a version of the top-secret deathstar plans to the *deathstar-plans* topic:
+
+::
+
+   $ echo "deathstar reactor design v3" | ./kafka-produce.sh --topic deathstar-plans
+
+This message shows up in the *empire-backup* window, but not for the outposts.
+
+Congratulations, Kafka is working as expected :)
+
+Step 5:  The Danger of a Compromised Kafka Client
+=================================================
+
+But what if a rebel spy gains access to any of the remote outposts that act as Kafka clients?
+Since every client has access to the Kafka broker on port 9092, it can do some bad stuff.
+For starters, the outpost container can actually switch roles from a consumer to a producer,
+sending "malicious" data to all other consumers on the topic.
+
+To prove this, kill the existing kafka-consume.sh command in the outpost-9999 window
+by typing control-C and instead run:
+
+::
+
+  $ echo "Vader Booed at Empire Karaoke Party" | ./kafka-produce.sh --topic empire-announce
+
+Uh oh!  Outpost-8888 and all of the other outposts in the empire have now received this fake announcement.
+
+But even more nasty from a security perspective is that the outpost container can access any topic
+on the kafka-broker.
+
+In the outpost-9999 container, run:
+
+::
+
+  $ ./kafka-consume.sh --topic deathstar-plans
+  "deathstar reactor design v3"
+
+We see that any outpost can actually access the secret deathstar plans.  Now we now how the rebels got
+access to them!
+
+Step 6: Securing Access to Kafka with Cilium
+============================================
+
+Obviously, it would be much more secure to limit each pod's access to the Kafka broker to be
+least privilege (i.e., only what is needed for the app to operate correctly and nothing more).
+
+We can do that with the following Cilium security policy.   As with Cilium HTTP policies, we can write
+policies that identify pods by labels, and then limit the traffic in/out of this pod.  In
+this case, we'll create a policy that identifies exactly the traffic that should be allowed to reach the
+Kafka broker, and deny the rest.
+
+As an example, a policy could limit containers with label *app=empire-outpost* to only be able to consume
+topic *empire-announce*, but would block any attempt by a compromised container (e.g., empire-outpost-9999)
+from producing to *empire-announce* or consuming from *deathstar-plans*.
+
+.. image:: images/cilium_kafka_gsg_attack.png
```

Comment:
If changing the pod name to be "Kafka Broker" in the earlier image, change here also for consistency.
Do we want a green "O" or a checkmark for the allowed Kafka-protocol rule "Consume" in the diagram?